test/acceptance/duplicate_tx_test.go

```diff
@@ -0,0 +1,85 @@
+package acceptance
+
+import (
+	"bytes"
+	"context"
+	"github.com/orbs-network/orbs-network-go/test/builders"
+	"github.com/orbs-network/orbs-network-go/test/harness"
+	"github.com/orbs-network/orbs-spec/types/go/protocol"
+	"github.com/stretchr/testify/require"
+	"testing"
+	"time"
+)
+
+func TestSendSameTransactionMoreThanOnce1(t *testing.T) {
+	harness.Network(t).AllowingErrors(
+		"error adding transaction to pending pool",
+		"error adding forwarded transaction to pending pool",
+		"error sending transaction",
+	).Start(func(ctx context.Context, network harness.TestNetworkDriver) {
+		ts := time.Now()
+
+		contract := network.GetBenchmarkTokenContract()
+		contract.DeployBenchmarkToken(ctx, 1)
+
+		// send three identical transactions to two nodes
+		network.SendTransactionInBackground(ctx, builders.TransferTransaction().WithTimestamp(ts).Builder(), 0)
```

Comment:
why not assign the built tx to a variable and then send the variable? it'll better signify the fact that it's _the same_ transaction

Reply:
because the interface accepts a builder and each builder can be used just once.